ONSdigital/census-rm-acceptance-tests-2021
# Receipting Scenario When AR < E

# Motivation and Context
Help show what process is for sending UPDATE to field when AR < E

# What has changed
Two new tests in receipting scenario with AR < E (note: it will not receipt but is still set to increment according to confluence table)

# How to test?
Run test in debug and check message on Field Rabbit queue
Load a sample and set the actualResponses/expected and check the variations send (or don't send) the correct output

# Links
https://trello.com/c/OEL6bfx5
https://collaborate2.ons.gov.uk/confluence/display/SDC/Receipting
